data_validation/__main__.py

```diff
@@ -183,6 +166,35 @@ def build_config_from_args(args, config_manager):
     Args:
         config_manager (ConfigManager): Validation config manager instance.
     """
+    if config_manager.validation_type == consts.CUSTOM_QUERY:
+        if args.custom_query_type is not None:
+            config_manager.append_custom_query_type(args.custom_query_type)
+        else:
+            raise ValueError(
+                "Expected custom query type to be given, got empty string."
+            )
+
+        if args.source_query_file is not None:
+            query_file = cli_tools.get_arg_list(args.source_query_file)
+            source_query_str = config_manager.get_query_from_file(query_file[0])
```

Comment:
In case of a typo or similar syntax error, the query will fail during execution. This is documented [here](https://docs.google.com/spreadsheets/d/1lwtblCauq83fnGb2JgiU-rJAD6una2BI5BWCwQ9bdqE/edit#gid=0&range=15:15)

We can possibly do a syntax check using sqlparse or similar libraries but I'm not sure if we should do it now or at a later stage. We can discuss this in our connect.